USERS.md

```diff
@@ -122,6 +122,12 @@ Users (Alphabetically)
       U: Networking, network policy, and network visibility
       L: https://cloud.google.com/blog/products/containers-kubernetes/bringing-ebpf-and-cilium-to-google-kubernetes-engine
 
+    * N: kOps
+      D: kOps is using Cilium as a one of the supported CNIs
```

Comment:
typo?
```suggestion
      D: kOps is using Cilium as one of the supported CNIs
```

Reply:
@olemarkus Do you have a chance to fix up the typo in your commit? Would be great to get it merged :) Thanks!

Reply:
Sorry I missed that. Done.